packages/common/src/intl/locales/en/replenishment.json

```diff
@@ -81,5 +81,6 @@
   "messages.deleted-requisitions_one": "Deleted {{count}} order",
   "messages.deleted-requisitions_other": "Deleted {{count}} orders",
   "messages.saved": "Saved",
-  "messages.select-rows-to-delete-them": "Select rows to delete them"
+  "messages.select-rows-to-delete-them": "Select rows to delete them",
+  "messages.delete-shipment-warning": "This will permanently delete shipment(s)"
```

Comment:
generally, I prefer using the plural function e.g. `label.number-months_one` and `label.number-months_other` in this file. The grammar of the singular version stands out for me.. 'This will permanently delete shipment' doesn't sound good.

Given how common this action / phrase is, maybe this should be in `common` and just be `This will delete permanently` or something suitably generic.

Reply:
Also - it is being used by Outbounds, so should be in `distribution`?